--- qudt.ttl	2023-04-09 13:02:07.109461100 +0200
+++ qudt.patched.ttl	2023-04-09 14:51:09.344104400 +0200
@@ -19,59 +19,6 @@
 @prefix voag: <http://voag.linkedmodel.org/schema/voag#> .
 @prefix xsd: <http://www.w3.org/2001/XMLSchema#> .
 
-dcterms:abstract
-  a owl:AnnotationProperty ;
-  rdfs:isDefinedBy dcterms: ;
-  rdfs:label "abstract" ;
-  rdfs:range xsd:string ;
-.
-dcterms:contributor
-  a rdf:Property ;
-  rdfs:label "contributor" ;
-.
-dcterms:created
-  a rdf:Property ;
-  rdfs:label "created" ;
-  rdfs:range xsd:date ;
-.
-dcterms:creator
-  a owl:AnnotationProperty ;
-  rdfs:label "creator" ;
-.
-dcterms:description
-  a owl:AnnotationProperty ;
-  rdfs:label "description" ;
-.
-dcterms:isReplacedBy
-  a owl:ObjectProperty ;
-  rdfs:label "is replaced by" ;
-.
-dcterms:modified
-  a rdf:Property ;
-  rdfs:label "modified" ;
-  rdfs:range xsd:date ;
-.
-dcterms:rights
-  a owl:AnnotationProperty ;
-  rdfs:label "rights" ;
-  rdfs:range xsd:string ;
-.
-dcterms:source
-  a owl:AnnotationProperty ;
-  rdfs:isDefinedBy dcterms: ;
-  rdfs:label "source" ;
-  rdfs:range xsd:anyURI ;
-.
-dcterms:subject
-  a owl:AnnotationProperty ;
-  rdfs:label "subject" ;
-  rdfs:range xsd:string ;
-.
-dcterms:title
-  a owl:AnnotationProperty ;
-  rdfs:label "title" ;
-  rdfs:range xsd:string ;
-.
 <http://qudt.org/2.1/schema/qudt>
   a owl:Ontology ;
   vaem:hasGraphMetadata vaem:GMD_QUDT-SCHEMA ;
@@ -3092,15 +3039,6 @@
   rdfs:isDefinedBy <http://qudt.org/2.1/schema/qudt> ;
   rdfs:label "QUDT Schema Catalog Entry" ;
 .
-voag:supersededBy
-  a owl:ObjectProperty ;
-  rdfs:isDefinedBy <http://voag.linkedmodel.org/schema/voag> ;
-  rdfs:label "superseded by" ;
-.
-<http://www.linkedmodel.org/schema/dtype>
-  vaem:namespace "http://www.linkedmodel.org/schema/dtype#"^^xsd:anyURI ;
-  vaem:namespacePrefix "dtype" ;
-.
 vaem:GMD_QUDT-SCHEMA
   a vaem:GraphMetaData ;
   dcterms:contributor "Daniel Mekonnen" ;
@@ -3168,9 +3106,3 @@
   rdfs:isDefinedBy <http://qudt.org/2.1/schema/qudt> ;
   rdfs:label "QUDT" ;
 .
-prov:wasDerivedFrom
-  a rdf:Property ;
-  rdfs:isDefinedBy <http://www.w3.org/ns/prov> ;
-  rdfs:label "was derived from" ;
-  rdfs:range qudt:Concept ;
-.
