--- SCHEMA_QUDT-DATATYPE-v2.1.ttl	2023-05-11 15:06:33.000000000 +0200
+++ SCHEMA_QUDT-DATATYPE-v2.1.patched.ttl	2023-05-11 21:58:05.109311400 +0200
@@ -21,39 +21,6 @@
 @prefix vaem: <http://www.linkedmodel.org/schema/vaem#> .
 @prefix voag: <http://voag.linkedmodel.org/schema/voag#> .
 @prefix xsd: <http://www.w3.org/2001/XMLSchema#> .
-
-dcterms:contributor
-  a owl:AnnotationProperty ;
-  rdfs:label "contributor" ;
-  rdfs:range xsd:string ;
-.
-dcterms:creator
-  a owl:AnnotationProperty ;
-  rdfs:label "creator" ;
-  rdfs:range xsd:string ;
-.
-dcterms:description
-  a owl:AnnotationProperty ;
-  a owl:DatatypeProperty ;
-  rdfs:isDefinedBy <http://qudt.org/2.1/schema/datatype> ;
-  rdfs:label "description" ;
-  rdfs:range rdf:HTML ;
-.
-dcterms:rights
-  a owl:AnnotationProperty ;
-  rdfs:label "rights" ;
-  rdfs:range xsd:string ;
-.
-dcterms:subject
-  a owl:AnnotationProperty ;
-  rdfs:label "subject" ;
-  rdfs:range xsd:string ;
-.
-dcterms:title
-  a owl:AnnotationProperty ;
-  rdfs:label "title" ;
-  rdfs:range xsd:string ;
-.
 <http://qudt.org/2.1/schema/datatype>
   a owl:Ontology ;
   vaem:hasCatalogEntry voag:QUDT-SchemaCatalogEntry ;
@@ -6328,75 +6295,6 @@
   rdfs:label "Z-Coordinate definition" ;
   rdfs:range xsd:string ;
 .
-dtype:GMD_DTYPE
-  dcterms:description "The ontology \"dtype\" provides a specification of simple data types such as enumerations and codelists. These are needed in support of the conversion of XML Schemas and UML Models to OWL. The purpose of \"dtype\" is to provide a foundation for expressing rich data types. These data types are typically needed when dealing with OWL representations of data structures derived from XML Schema." ;
-.
-vaem:CollectionGraph
-  dcterms:description "A \"vaem:CollectionGraph\" is a graph that aggregates other graphs using \"owl:imports\" statements." ;
-.
-vaem:ConfigurationGraph
-  dcterms:description "A graph that serves as a registy or provides other parameter settings." ;
-.
-vaem:ConnectSetGraph
-  dcterms:description "A graph that specifies mapings between external data sources and a neutral model." ;
-.
-vaem:ConstraintsGraph
-  dcterms:description "A graph that is for defining constraints using SPIN." ;
-.
-vaem:ControllersGraph
-  dcterms:description "A graph that acts as a controller in an MVC-like framework." ;
-.
-vaem:CurationGraph
-  dcterms:description "A graph to hold metadata that is needed to describe, govern and provision another graph. A curation graph will likely use VOAG and VAEM for this purpose. The property \"vaem:curationGraph\" is used to link to the graph (or graphs) being curated" ;
-.
-vaem:DataGraph
-  dcterms:description "A graph that holds data instances." ;
-.
-vaem:DocumentationGraph
-  dcterms:description "A graph that is soley for the purpose of generating documentation." ;
-.
-vaem:FunctionsGraph
-  dcterms:description "A graph that defines SPIN functions." ;
-.
-vaem:GMD_VAEM
-  dcterms:description "VAEM stands for \"Vocabulary for Attaching Essential Metadata\". What VAEM regards as essential metadata is data about dates and times, confidentiality, and other characterisitic qualifiers of the ontology, but also references to where a ontology is documented and where to find ontology registration for governance, attribution and provenance. VAEM makes use of some properties from the DC terms vocabulary using the namespace prefix \"dct\"." ;
-.
-vaem:LinkSetGraph
-  dcterms:description "A graph that holds statements that specify links between resources." ;
-.
-vaem:MappingGraph
-  dcterms:description "A graph that specifies a transformation of a graph into another graph or graphs. Invariably this is a SPINMap graph using SPARQL Rules (SPIN) to express the transforms." ;
-.
-vaem:ProxyGraph
-  dcterms:description "A graph that represents an external data source, or schema, as an ontology model." ;
-.
-vaem:ReferenceDataGraph
-  dcterms:description "A graph that holds reference data." ;
-.
-vaem:RulesGraph
-  dcterms:description "A graph that holds SPIN rules." ;
-.
-vaem:SchemaGraph
-  dcterms:description "A graph that defines classes and properties for one or more namespaces." ;
-.
-vaem:ScriptGraph
-  dcterms:description "A graph that defines SWP templates that are of a general scripting nature." ;
-.
-vaem:ServicesGraph
-  dcterms:description "A graph that implements SWP services." ;
-.
-vaem:TransformsGraph
-  dcterms:description "A graph that uses SPIN to define transforms." ;
-.
-vaem:ViewGraph
-  dcterms:description "A graph that provides the access to services in an MVC-like framework." ;
-.
-vaem:VocabularyGraph
-  dcterms:description "A graph that is a controlled set of instances that define a terminology. Typically this is based on SKOS." ;
-.
-xsd:dateTime
-  dcterms:description "YYYY-MM-DDThh:mm:ss[.dd] or YYYY-DDDThh:mm:ss[.dd], where 'YYYY' is the year, 'MM' is the two-digit month, 'DD' is the two-digit day, 'DDD' is the three digit day of year, 'T' is constant, 'hh:mm:ss[.dd]' is the UTC time in hours, minutes, seconds, and optional fractional seconds. As many 'd' characters to the right of the period as required may be used to obtain the required precision. All fields require leading zeros." ;
-.
 [
   a owl:Restriction ;
   owl:allValuesFrom qudt:ScalarDatatype ;
