--- foaf.ttl	2023-04-22 14:21:57.826663900 +0200
+++ foaf.patched.ttl	2023-05-16 12:44:06.284528800 +0200
@@ -1,7 +1,6 @@
 @prefix foaf: <http://xmlns.com/foaf/0.1/> .
 @prefix owl: <http://www.w3.org/2002/07/owl#> .
 @prefix dc11: <http://purl.org/dc/elements/1.1/> .
-@prefix wot: <http://xmlns.com/wot/0.1/> .
 @prefix rdfs: <http://www.w3.org/2000/01/rdf-schema#> .
 @prefix ns0: <http://www.w3.org/2003/06/sw-vocab-status/ns#> .
 @prefix schema: <http://schema.org/> .
@@ -14,14 +13,6 @@
   a owl:Ontology ;
   dc11:title "Friend of a Friend (FOAF) vocabulary" ;
   dc11:description "The Friend of a Friend (FOAF) RDF vocabulary, described using W3C RDF Schema and the Web Ontology Language." .
-
-wot:assurance a owl:AnnotationProperty .
-wot:src_assurance a owl:AnnotationProperty .
-<http://www.w3.org/2003/06/sw-vocab-status/ns#term_status> a owl:AnnotationProperty .
-dc11:description a owl:AnnotationProperty .
-dc11:title a owl:AnnotationProperty .
-dc11:date a owl:AnnotationProperty .
-rdfs:Class a owl:Class .
 foaf:LabelProperty
   a rdfs:Class, owl:Class ;
   ns0:term_status "unstable" ;
@@ -46,10 +37,6 @@
   rdfs:comment "An agent (eg. person, group, software or physical artifact)." ;
   owl:equivalentClass dc:Agent .
 
-geo:SpatialThing
-  a owl:Class ;
-  rdfs:label "Spatial Thing" .
-
 foaf:Document
   a rdfs:Class, owl:Class ;
   rdfs:label "Document" ;
@@ -106,8 +93,6 @@
   rdfs:comment "An online account." ;
   rdfs:isDefinedBy foaf: ;
   rdfs:subClassOf owl:Thing .
-
-owl:Thing rdfs:label "Thing" .
 foaf:OnlineGamingAccount
   a rdfs:Class, owl:Class ;
   ns0:term_status "unstable" ;
@@ -587,8 +572,6 @@
   rdfs:domain skos:Concept ;
   rdfs:range owl:Thing ;
   rdfs:isDefinedBy foaf: .
-
-skos:Concept rdfs:label "Concept" .
 foaf:isPrimaryTopicOf
   a rdf:Property, owl:InverseFunctionalProperty ;
   ns0:term_status "stable" ;
