--- SCHEMA_QUDT_NoOWL-v2.1.ttl	2023-04-13 23:41:04.083203400 +0200
+++ SCHEMA_QUDT_NoOWL-v2.1.patched.ttl	2023-04-13 23:51:45.793659000 +0200
@@ -21,53 +21,6 @@
 @prefix vaem: <http://www.linkedmodel.org/schema/vaem#> .
 @prefix voag: <http://voag.linkedmodel.org/schema/voag#> .
 @prefix xsd: <http://www.w3.org/2001/XMLSchema#> .
-
-dcterms:abstract
-  a rdf:Property ;
-  rdfs:isDefinedBy <http://qudt.org/2.1/schema/shacl/qudt> ;
-  rdfs:label "abstract" ;
-.
-dcterms:contributor
-  a rdf:Property ;
-  rdfs:label "contributor" ;
-.
-dcterms:created
-  a rdf:Property ;
-  rdfs:label "created" ;
-.
-dcterms:creator
-  a rdf:Property ;
-  rdfs:label "creator" ;
-.
-dcterms:description
-  a rdf:Property ;
-  rdfs:label "description" ;
-.
-dcterms:isReplacedBy
-  a rdf:Property ;
-  rdfs:label "is replaced by" ;
-.
-dcterms:modified
-  a rdf:Property ;
-  rdfs:label "modified" ;
-.
-dcterms:rights
-  a rdf:Property ;
-  rdfs:label "rights" ;
-.
-dcterms:source
-  a rdf:Property ;
-  rdfs:isDefinedBy <http://qudt.org/2.1/schema/shacl/qudt> ;
-  rdfs:label "source" ;
-.
-dcterms:subject
-  a rdf:Property ;
-  rdfs:label "subject" ;
-.
-dcterms:title
-  a rdf:Property ;
-  rdfs:label "title" ;
-.
 <http://qudt.org/2.1/schema/shacl/qudt>
   a owl:Ontology ;
   vaem:hasGraphMetadata vaem:GMD_SHACLQUDT-SCHEMA ;
@@ -3336,15 +3289,6 @@
   rdfs:isDefinedBy <http://qudt.org/2.1/schema/shacl/qudt> ;
   rdfs:label "QUDT Schema Catalog Entry" ;
 .
-voag:supersededBy
-  a rdf:Property ;
-  rdfs:isDefinedBy <http://qudt.org/2.1/schema/shacl/qudt> ;
-  rdfs:label "superseded by" ;
-.
-<http://www.linkedmodel.org/schema/dtype>
-  vaem:namespace "http://www.linkedmodel.org/schema/dtype#"^^xsd:anyURI ;
-  vaem:namespacePrefix "dtype" ;
-.
 vaem:GMD_SHACLQUDT-SCHEMA
   a vaem:GraphMetaData ;
   dcterms:contributor "Daniel Mekonnen" ;
@@ -3412,7 +3356,3 @@
   rdfs:isDefinedBy <http://qudt.org/2.1/schema/shacl/qudt> ;
   rdfs:label "QUDT" ;
 .
-prov:wasDerivedFrom
-  a rdf:Property ;
-  rdfs:label "was derived from" ;
-.
