--- vann.ttl	2023-04-09 13:02:07.141038300 +0200
+++ vann.patched.ttl	2023-04-09 13:46:22.108533900 +0200
@@ -18,20 +18,6 @@
   ns0:preferredNamespaceUri "http://purl.org/vocab/vann/" ;
   ns0:preferredNamespacePrefix "vann" .
 
-<http://iandavis.com/id/me>
-  a foaf:Person ;
-  foaf:name "Ian Davis" .
-
-<https://vocab.org/vann/vann-vocab-20100607.rdf>
-  a <http://web.resource.org/cc/Work> ;
-  ns1:license <http://creativecommons.org/licenses/by/1.0/> ;
-  dc:type <http://purl.org/dc/dcmitype/Text> .
-
-<http://creativecommons.org/licenses/by/1.0/>
-  a ns1:License ;
-  ns1:permits ns1:Reproduction, ns1:Distribution, ns1:DerivativeWorks ;
-  ns1:requires ns1:Notice, ns1:Attribution .
-
 ns0:changes
   a owl:AnnotationProperty ;
   rdfs:label "Changes"@en ;
