--- vann.ttl	2023-04-22 14:21:57.689839100 +0200
+++ vann.patched.ttl	2023-05-16 16:22:39.142365300 +0200
@@ -1,8 +1,6 @@
 @prefix owl: <http://www.w3.org/2002/07/owl#> .
 @prefix dc: <http://purl.org/dc/terms/> .
 @prefix ns0: <http://purl.org/vocab/vann/> .
-@prefix foaf: <http://xmlns.com/foaf/0.1/> .
-@prefix ns1: <http://web.resource.org/cc/> .
 @prefix rdfs: <http://www.w3.org/2000/01/rdf-schema#> .
 
 <http://purl.org/vocab/vann/>
@@ -18,20 +16,6 @@
   ns0:preferredNamespaceUri "http://purl.org/vocab/vann/" ;
   ns0:preferredNamespacePrefix "vann" .
 
-<http://iandavis.com/id/me>
-  a foaf:Person ;
-  foaf:name "Ian Davis" .
-
-<https://vocab.org/vann/vann-vocab-20100607.rdf>
-  a <http://web.resource.org/cc/Work> ;
-  ns1:license <http://creativecommons.org/licenses/by/1.0/> ;
-  dc:type <http://purl.org/dc/dcmitype/Text> .
-
-<http://creativecommons.org/licenses/by/1.0/>
-  a ns1:License ;
-  ns1:permits ns1:Reproduction, ns1:Distribution, ns1:DerivativeWorks ;
-  ns1:requires ns1:Notice, ns1:Attribution .
-
 ns0:changes
   a owl:AnnotationProperty ;
   rdfs:label "Changes"@en ;
